lib/common.sh

```diff
@@ -144,6 +144,7 @@ export CAPM3PATH="${CAPM3PATH:-${M3PATH}/cluster-api-provider-metal3}"
 export CAPM3_BASE_URL="${CAPM3_BASE_URL:-metal3-io/cluster-api-provider-metal3}"
 export CAPM3REPO="${CAPM3REPO:-https://github.com/${CAPM3_BASE_URL}}"
 export CAPM3RELEASEBRANCH="${CAPM3RELEASEBRANCH:-main}"
+export BMORELEASEBRANCH="${BMORELEASEBRANCH:-main}"
```

Comment:
I'm not sure we should set this default. Below we set `BMOBRANCH` based on this variable or a default if unset, but since we set this already here, the default will never be used. Do we need to set it here?

Reply:
good point, probably not